media/com_associations/js/admin-autoassoc-modal.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+(function() {
+  'use strict';
+
+  document.addEventListener('DOMContentLoaded', function() {
+    var applyBtn = document.getElementById('applyBtn');
+
+    applyBtn.addEventListener('click', function(e) {
+      e.preventDefault();
+      var checkedBoxes = document.querySelectorAll("td.row-selected input[type='checkbox']");
+      var languageIds = [];
+
+      checkedBoxes.forEach(function(box) {
```

Comment:
forEach will not work with NodeList on Microsoft edge and IE.
Replace ` var checkedBoxes = document.querySelectorAll("td.row-selected input[type='checkbox']");`
with
` var checkedBoxes = [].slice.call(document.querySelectorAll("td.row-selected input[type='checkbox']"));`

Reply:
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/10638731/